### PR TITLE
[Example App] Remove old example App and add basic new one

### PR DESCRIPTION
**Pull request by mpiechocki**
Removed old example app and added a blank new one. Got rid of the `Main.storyboard` ;)

### CODE REVIEW

**Review by minixT — CHANGES_REQUESTED** [review]
Reviewed. Looks good. Please see my comments and let me know what will be the part of this PR and what you will change in the next steps.

File: ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            .flatMap { $0.scanForPeripherals(withServices: [serviceUuid]) }
+            .take(1)
+            .flatMap { $0.peripheral.establishConnection() }
+            .do(onNext: { [weak self] _ in self?.centralView.readValueLabel.isEnabled = true })
+            .flatMap { $0.discoverServices([serviceUuid]) }
+            .flatMap { Observable.from($0) }
+            .flatMap { $0.discoverCharacteristics([characteristicUuid]) }
+            .flatMap { Observable.from($0) }
+            .flatMap { $0.observeValueUpdateAndSetNotification() }
+            .subscribe(onNext: { [weak self] in
```
[the remark this answers is not in the document]
Comment (mpiechocki) [reply ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift:59]: I think I'll add a basic, generic way to handle all errors (like an alert just saying what's the error). For me, the purpose of this app should be to show how to use the library in a simple way, without too much clutter. What do you think?

File: ExampleApp/ExampleApp/Screens/Peripheral/PeripheralViewController.swift [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    // MARK: - Private
+
+    private let disposeBag = DisposeBag()
+    private lazy var manager = PeripheralManager()
+    private var characteristic: CBMutableCharacteristic?
+
+    @objc private func handleAdvertiseButton() {
+        guard let serviceUuidString = peripheralView.serviceUuidTextField.text,
+              let characteristicUuidString = peripheralView.characteristicUuidTextField.text,
+              let value = peripheralView.valueTextField.text else { return }
```
[the remark this answers is not in the document]
Comment (mpiechocki) [reply ExampleApp/ExampleApp/Screens/Peripheral/PeripheralViewController.swift:39]: Actually, I think we should just leave it as simple as can be (and maybe add a note in readme, that you have to enter a proper uuid strings). Again, I think we should keep the app as simple as can be, believing that people running this app would be just developers trying to get to know how to use the lib :)

File: ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (minixT) [reply ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift:59]: I think simple alert will be enough.

File: ExampleApp/ExampleApp/Screens/Peripheral/PeripheralViewController.swift [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        )
+    }
+
+    private func startAdvertising(service: CBMutableService) {
+        let managerIsOn = manager.observeStateWithInitialValue()
+            .filter { $0 == .poweredOn }
+
+        Observable.combineLatest(managerIsOn, Observable.just(manager)) { $1 }
+            .flatMap { $0.add(service) }
+            .flatMap { [manager] in manager.startAdvertising($0.advertisingData) }
```
[the remark this answers is not in the document]
Comment (mpiechocki) [reply ExampleApp/ExampleApp/Screens/Peripheral/PeripheralViewController.swift:79]: I beleive so :)
both `PeripheralViewController` the closure have the reference to manager, so there's no cycle. Quick testing with Memory Graph tool proves it :)

File: ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (mpiechocki) [reply ExampleApp/ExampleApp/Screens/Central/CentralViewController.swift:59]: I'll add a generic alert in this PR

**Comment by mpiechocki** [comment]
@minixT fyi, I pushed some commits with your suggestions :)

**Review by minixT — APPROVED** [review]

**Pull request closed without merging**